odl/operator/default_ops.py

```diff
@@ -366,7 +369,118 @@ def __repr__(self):
 
     def __str__(self):
         """Return ``str(self)``."""
-        return "x * y"
+        return "x * {}".format(self.y)
+
+
+class PowerOperator(Operator):
+
+    """Take power of vector or scalar.
+
+    ``MultiplyOperator(n)(x) <==> x ** n``
+
+    Here, ``x`` is a `LinearSpaceVector` or `Field` element and
+    ``y`` is a number.
+    Hence, this operator can be defined either on a `LinearSpace` or on
+    a `Field`.
+    """
+
+    def __init__(self, domain, exponent):
+        """Initialize a MultiplyOperator instance.
+
+        Parameters
+        ----------
+        exponent : Number
+            The power to take
+        domain : `LinearSpace` or `Field`, optional
+            The set to take values in
+        """
+
+        self.exponent = float(exponent)
+        self._domain_is_field = isinstance(domain, Field)
```

Comment:
You're storing this, so I was wondering if you consider this to be bogging down code. It's basically used once per call only. Storing makes it static, and I think we should only store if there is a good reason. Dynamic properties are preferable from a subclassing point of view.